cluster.js

```diff
@@ -42,9 +42,16 @@ const _             = require('lodash');
 const StreamCounter = require('./stream-counter.js').StreamCounter;
 const L7mpOpenAPI   = require('./l7mp-openapi.js').L7mpOpenAPI;
 const utils         = require('./stream.js');
+const listener      = require('./listener');
 const Status        = utils.Status;
 // for getAtPath()
 const Rule          = require('./rule.js').Rule;
+//for prometheus
+const client        = require('prom-client');
+const monitoring    = require('./monitoring.js').Monitoring
+const cluster_requests_total = require('./monitoring.js').cluster_requests_total;
+const session_byte_counter_total = require('./monitoring').session_byte_counter_total;
+const session_packet_counter_total = require('./monitoring').session_packet_counter_total;
```

Comment:
I guess this will not quite scale when we'll have dozens of metrics but it's going to be fine for now

Reply:
True, it won't. You can find the changes in the latest commits.